Src/Plugins/Tests.Security/TestData/UseSecureApi/ExpectedOutputs/SEC104.Memory.Allocation_alloca.sarif

```diff
@@ -73,11 +73,10 @@
             }
           ],
           "fingerprints": {
-            "AssetFingerprint/v1": "",
-            "ValidationFingerprint/v1": "[secret=D62A0E587857E47344ED096CC889828D53B1AB3EA83E005D277AC18A2AB80B4F]",
-            "ValidationFingerprintHash/v1": "b344d33a3519f593c86ad5274421711b348eb2b250a69bdb4b6d3600bc069563"
-          },
-          "rank": 54.99
+            "AssetFingerprint/v1": "[id=_alloca]",
```

Comment:
>_alloca [](http://example.com/codeflow?start=40&length=7)

use secure api rule sets the relevant api to 'id'. this data isn't considered a secret...